bci_tester/data.py

```diff
@@ -393,6 +399,38 @@ def create_BCI(
     BUSYBOX_CONTAINER,
 ]
 
+#: Metadata labels according to https://confluence.suse.com/display/ENGCTNRSTORY/SLE+BCI+Image+Overview
+#: Containers that are in tech preview
+TECHPREVIEW_CONTAINERS = [
```

Comment:
Rust is missing.

I would suggest to add an assertion to the metadata test that `len(TECHPREVIEW_CONTAINERS) + len(L3_CONTAINERS) == len(ALL_CONTAINERS)`

Reply:
Dammit, yes. I've added the rust containers by version manually because 1.63 will be in L3.

That assertion will not work, because e.g. golang 1.16 and 1.17 are not in techpreview nor in i3, at least not according to the reference sheet in confluence.

Reply:
Everything that is not in l3 is techpreview. unsupported just means eol in that table, but the label will still be techpreview.

Reply:
I changed the PR such, that it now checks all containers in `L3_CONTAINERS` for the l3 label, all other containers need to have the `techpreview` label. This is simpler, does not require additional checks to ensure that we are not missing containers and should work as well.